modules/dynamic/test/src-jvm/smithy4s/dynamic/OperationSpec.scala

```
@@ -18,31 +18,12 @@ package smithy4s
 package dynamic
 
 import model._
-import software.amazon.smithy.model.{Model => SModel}
-import software.amazon.smithy.model.shapes.ModelSerializer
-import java.nio.file.Paths
-import cats.syntax.all._
 import http.HttpEndpoint
-import DummyIO._
 
-class OperationSpec() extends munit.FunSuite {
-
-  // This is not ideal, but it does the job.
-  val cwd = System.getProperty("user.dir");
-  val pizzaSpec = Paths.get(cwd + "/sampleSpecs/pizza.smithy").toAbsolutePath()
+class OperationSpec() extends DummyIO.Suite {
 
   test("Decode operation") {
-    IO(
```

`check` no longer required (due to `DummyIO.Suite`). Here it was not called at all, but test was silently failing (missing `alloy` dependency).

---

good catch